hermeto/core/package_managers/pip.py

```diff
@@ -28,6 +27,7 @@
     Union,
     cast,
 )
+from urllib import parse
```

Comment:
Probably better to namespace this as `from urllib import parse as urlparse`. Apologies for having forgotten to mention it in the previous round of review.